src/OpenLoco/Config.h

```diff
@@ -88,7 +88,7 @@ namespace OpenLoco::Config
         ObjectHeader preferredCurrency;                 // 0x7C
         uint8_t enabledMusic[29];                       // 0x50AF40, 0x8C
         uint8_t pad_A9[0xCC - 0xA9];                    // 0xA9
-        int32_t volume;                                 // 0x50AF80, 0xCC
+        float volume;                                   // 0x50AF80, 0xCC // originally int32_t
```

Comment:
unsure if this kind of abuse it allowed...

Reply:
Let's switch over to NewConfig instead, yeah.

Reply:
Roger that, I think NewConfig is better anyways since all the settings for other channel volume sliders will live there too, so it unified those settings when I make that PR